### PR TITLE
nitter: unstable-2023-04-21 -> unstable-2023-05-19

### PR DESCRIPTION
**Pull request by Infinidoge**
###### Description of changes

Trivial bump of Nitter.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: pkgs/servers/nitter/default.nix [hunk 1]
```diff
@@ -7,13 +7,13 @@
 
 nimPackages.buildNimPackage rec {
   pname = "nitter";
-  version = "unstable-2023-04-21";
+  version = "unstable-2023-05-19";
```
Comment (erdnaxe) [inline pkgs/servers/nitter/default.nix:10]:
```suggestion
  version = "unstable-2023-05-20";
```
(according to https://github.com/zedeus/nitter/commit/e3b3b38a2d43a83b5fc2239ab41e864ee686fb2f)

**Review by erdnaxe — APPROVED** [review]
LGTM, this is a small bump.

File: pkgs/servers/nitter/default.nix (hunk 1 above)
Comment (erdnaxe) [reply pkgs/servers/nitter/default.nix:10]: This now makes more sense, thanks :O

With a browser in UTC time, it is showing "May 20, 2023, 12:10 AM UTC".
This is a small detail, we can merge this in EDT timezone unless there is a explicit policy in NixPkgs to use UTC timezone?

**Comment by ehmry** [comment]
I don't think the date will matter between now and the next bump.

**Pull request merged**